src/styles/main.scss

```diff
@@ -1,3 +1,55 @@
 body {
   margin: 0;
+
+  --height-width-of-block: 100px;
+}
+
+$num-days: 31;
+
+@for $i from 1 through $num-days {
+  .day-#{$i} {
+    font-family: Arial, sans-serif;
+    box-sizing: border-box;
+    height: var(--height-width-of-block);
+    width: var(--height-width-of-block);
+    background-color: #eee;
+    border: 1px black solid;
+    text-align: center;
+    line-height: var(--height-width-of-block);
+    font-size: 30px;
+    transition: all 0.5s ease;
```

Comment:
it's not a good practice to use `all` as transition property. Specify what exactly you'd like to change